### ISSUE TITLE
Unprivileged Singularity is no longer running properly in user namespace

### ISSUE THREAD

**Issue by GodloveD** [apptainer/singularity#1546]
## Version of Singularity:

2.5.1

### Expected behavior

```
$ singularity shell some_sandbox
Singularity: Invoking an interactive shell within container...

Singularity>
```

### Actual behavior

```
$ singularity shell some_sandbox
ERROR  : Failed to resolve path to /usr/local/var/singularity/mnt/container
ABORT  : Retval = 255
```

### Steps to reproduce behavior

Install Singularity 2.5.1 on a kernel that supports user namespaces and try to shell into a sandbox.

**Comment by jmstover** [comment]
Err... I don't have that issue:

```
$ uname -r
4.4.14
$ /usr/local/singularity/2.5.1-nosuid/bin/singularity shell debian
Singularity: Invoking an interactive shell within container...

Singularity debian:~/tmp/sing> exit
```

Was configured with:
`./configure --prefix=/usr/local/singularity/2.5.1-nosuid --disable-suid`

**Comment by GodloveD** [comment]
Ahh.  This is actually already fixed in the release-2.5 branch (indicating that it will be fixed in next release) awesome.  Thanks for the reminder @jmstover and @cclerget.  Closing.

**Comment by DrDaveD** [comment]
I also do not see this issue with the released 2.5.1 code running on RHEL7 with unprivileged user namespaces enabled.  @godloved I am curious how you compiled 2.5.1 when you saw this problem.
